js_parser/load_es_grammar.py

```diff
@@ -45,9 +45,11 @@ def load_lexical_grammar(filename):
 ECMASCRIPT_SYNTHETIC_TERMINALS = {
     'IdentifierName': OrderedSet([
         'Name',
+        'BooleanLiteral',
+        'NullLiteral',
         *ECMASCRIPT_FULL_KEYWORDS,
         *ECMASCRIPT_CONDITIONAL_KEYWORDS
-    ]),
+    ]) - OrderedSet(['true', 'false', 'null']),
```

Comment:
might be nice to add comment why `BooleanLiteral`/`NullLiteral` not `true`/`false`/`null`

Reply:
Good idea, added.